core/modules/user/user.module

```diff
@@ -1772,13 +1791,49 @@ function user_page_title($account) {
   return is_object($account) ? user_format_name($account) : '';
 }
 
+/**
+ * Implements hook_init().
+ */
+function user_init() {
+  $current_path = current_path();
+  $user_login_page_settings = user_login_page_settings();
+  if (!user_is_logged_in() && ($current_path == 'user/login' || $current_path == 'user/password' || $current_path == 'user/register')) {
+    if ($user_login_page_settings == 'simplified') {
```

Comment:
I'm not going to make a suggestion everywhere, but if renamed here's what these would look like:

```suggestion
  $user_login_appearance = user_login_appearance_setting();
  if (!user_is_logged_in() && ($current_path == 'user/login' || $current_path == 'user/password' || $current_path == 'user/register')) {
    if ($user_login_appearance == 'simplified') {
```

Reply:
Done